lib/internal/bootstrap/node.js

```diff
@@ -39,6 +39,33 @@ const { JSON, Object, Symbol } = primordials;
 const config = internalBinding('config');
 const { deprecate } = require('internal/util');
 
+{
+  const { setPrepareStackTraceCallback } = internalBinding('errors');
+  const MainContextError = Error;
+  const ErrorToString = Error.prototype.toString;
+  setPrepareStackTraceCallback((globalThis, error, trace) => {
```

Comment:
Would be cleaner if the callback itself is implemented in e.g. `internal/errors.js` and just get imported here so we move the details out of the way.